test/plugins/test_lyrics.py

```diff
@@ -26,14 +26,15 @@
 
 from beets.library import Item
 from beets.test.helper import PluginMixin, TestHelper
-from beetsplug import lyrics
 
 from .lyrics_pages import LyricsPage, lyrics_pages
 
 github_ci = os.environ.get("GITHUB_ACTIONS") == "true"
```

Comment:
I think we'd rather **always** run these tests locally. We can avoid having to install all extra dependencies by adding `langdetect` to test dependencies under

```toml
[tool.poetry.group.test.dependencies]
```

This way `poetry install` will include this dependency. I'd suggest to also remove this block of code